src/instruments/src/ISIS/ArtificialHorizon.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { useSimVar } from '@instruments/common/simVars';
+import { PitchScale } from './PitchScale';
+import { RollScale } from './RollScale';
+import { RollIndex } from './RollIndex';
+
+const Sky: React.FC = () => <rect x={-256} y={-512} width={1024} height={768} className="sky" />;
+const Earth: React.FC = () => <rect x={-256} y={256} width={1024} height={768} className="earth" />;
+
+export const ArtificialHorizon: React.FC<{ maskWidth: number }> = ({ maskWidth }) => {
+    const [pitch] = useSimVar('PLANE PITCH DEGREES', 'degrees');
+    const [roll] = useSimVar('PLANE BANK DEGREES', 'degrees');
+    const [isOnGround] = useSimVar('SIM ON GROUND', 'Bool');
+
+    const pitchDegPixels = (512 - 2 * maskWidth) / 37;
+
+    const pitchShift = -pitch * pitchDegPixels;
+
+    return (
+        <g id="ArtificialHorizon">
+            <g id="RollGroup" transform={`rotate(${roll} 256 256)`}>
+                <g id="PitchGroup" transform={`translate(0 ${pitchShift})`}>
+                    <Sky />
+                    <Earth />
```

Comment:
I don't see the need for those to be components. The element itself with a comment is sufficient, and wastes less resources.